addons/website/static/src/js/menu/content.js

```diff
@@ -645,8 +649,8 @@ var EditMenuDialog = weWidgets.Dialog.extend({
                     'to_delete': this.toDelete,
                 }
             ],
-        }).then(function () {
-            return _super();
+        }).then(() => {
+            return _super(args);
```

Comment:
- There you don't transfer all arguments, you transform one argument which is a list of all the arguments... you need to use the spread operator
- You don't need to use a `args` temporary variable

=> `return _super(...arguments)`

Reply:
Oups, sorry for forgetting to use the spread operator... I thought that arrow functions had the arguments local variable as well, I discover that they do not. Thanks !